target/snitch_cluster/sw/riscv-tests/riscv-tests.mk

```diff
@@ -0,0 +1,129 @@
+# Copyright 2025 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Pascal Etterli <petterli@student.ethz.ch>
+#
+# This Makefile builds the ISA tests from the RISC-V Test repository.
+# It makes use of the ETHZ LLVM clang infrastructure instead of the provided
+# gcc build setup in the original repository.
+# See also: https://github.com/riscv-software-src/riscv-tests
+#
+# It ignores following tests as we cannot build them with the snitch
+# compiler toolchain.
+# - rv32ua: specified immediate (rv32ua/../rv64ua/amomaxu_w.S:34:55) exceeds integer range
+# - rv32uf: The qNaN #define uses the '0f:' specifier which is not recognized by our LLVM.
+# - rv32ud: same as rv32uf, but with '0d:'.
+# - rv32mi: CSR tcontrol is not recognized
+#
+# It ignores the following test cases as we don't implement the extensions
+# - rv32uc
+# - rv32si
+# - rv32uzfh
+# - rv32uzba
+# - rv32uzbb
+# - rv32uzbc
+# - rv32uzbs
+
```

Comment:
Move this entire comment block to the point where the Makefrags are included below, to make sure that the comment is always updated together with the include directives.